presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaPageSource.java

```diff
@@ -276,70 +275,123 @@ private void buildPagesEnabledRoles()
         }
     }
 
-    private void buildPagesColumns(QualifiedTablePrefix prefix)
+    private void buildPagesColumns(QualifiedTablePrefix schemaPrefix)
```

Comment:
nit: since this method does not necessarily "build" pages, how about a different name? like `addColumnsRecordsFor`, `addTablesRecordsFor` etc